packages/storybook/stories/CallComposite/CallCompositeDocs.tsx

```diff
@@ -68,6 +69,9 @@ export const getDocs: () => JSX.Element = () => {
         can be set at any time and immediately updates the composite UI to be optimized for a mobile device.
       </Description>
       <Source code={formFactorSnippet} />
+      <MessageBar>
+        Note: Only Protrait mode is supported when the `formFactor` is set to "mobile". Landscape mode is not supported.
```

Comment:
I like this better, much clearer, can you remove the line a couple lines above that says "Currently the mobile form factor only supports Portrait orientation and
        not Landscape."

And also apply this to the meeting composite docs